SECURITY.md

```diff
@@ -0,0 +1,38 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of our project are actively supported with security updates:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | ✅ Fully supported |
+| 5.0.x   | ❌ Unsupported     |
+| 4.0.x   | ✅ Supported (LTS)|
+| < 4.0   | ❌ Unsupported     |
+
+🔒 We generally provide security updates for the **latest major version** and the **most recent LTS version**.
```

Comment:
At this point, the only thing we can realistically commit to is that the current release will receive security updates. It hasn’t been decided yet whether there will be an additional LTS branch. To avoid overpromising or publishing information that isn’t true today, I recommend simplifying the “Supported Versions” section to reflect the current release only. We can always expand this later once an LTS strategy is formally agreed.

Reply:
done